### PR TITLE
Implemented test cases for session in start page

### PR DESCRIPTION
**Pull request by waseem-omar**

### CODE REVIEW

**Comment by waseem-omar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;jSkX1Old&#x2F;380-register-create-test-cases-for-session-status-on-start-page-for-widget-brand-reg-source-requests"> Register - Create test cases for session status on Start page for widget&#x2F;brand&#x2F;reg_source requests.</a>

File: widget/migrations/0004_auto_20151125_1011.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.core.validators
+
+
+class Migration(migrations.Migration):
```
Comment (danaspiegel) [inline widget/migrations/0004_auto_20151125_1011.py]: @waseem-omar do you know why this migration is included in this branch even though there's no model changes?

Comment (waseem-omar) [inline widget/migrations/0004_auto_20151125_1011.py]: @danaspiegel This is here by mistake, because when I wrote the test cases I created its branch that doesn't have the widget_host_url field changed to CharField instead of URLField. So I changed it for this pull request and seems that I reverted the change in models.py but didn't revert this change. So This file shouldn't be merged.

**Comment by danaspiegel** [comment]
@waseem-omar can you please fix the branch to remove the migration?

**Comment by waseem-omar** [comment]
@danaspiegel I took a look into this and found that we updated the host_url field to be a CharField instead of URLField, but the migration for that change was not created at that time. So that is why the migration is here now.
